pkg/core/portal.go

```diff
@@ -394,6 +398,61 @@ func (p *AuthPortal) ServeHTTP(w http.ResponseWriter, r *http.Request, upstreamO
 	}
 }
 
+// supplementClaims copied from the oauth2 backend.
+func (p *AuthPortal) supplementClaims(claims *jwtclaims.UserClaims) {
```

Comment:
@SvenDowideit , this is a partial copy of `func (b *Backend) supplementClaims(claims *jwtclaims.UserClaims) {`. shared function?

Reply:
yup to both - this was really just an mvp to see if this approach would work - the previous idea didn't pan out :)

Reply:
removed duplication :)